### PR TITLE
Discord app not found error & args check

### PR DESCRIPTION
**Pull request by spinfal**
Resolves https://github.com/replugged-org/replugged/issues/105

### CODE REVIEW

File: injectors/index.js [hunk 1]
```diff
@@ -30,11 +30,18 @@ try {
 const VALID_PLATFORMS = [ 'stable', 'ptb', 'canary', 'dev', 'development' ];
 
 (async () => {
-  let platform = (
-    process.argv.find(x => VALID_PLATFORMS.includes(x.toLowerCase())) || 'stable'
-  ).toLowerCase();
-  if (platform === 'development') {
+  let platform = process.argv.find(x => VALID_PLATFORMS.includes(x.toLowerCase()));
```
Comment (asportnoy) [inline injectors/index.js:33]: No argument should still default to stable.

File: injectors/index.js [hunk 2]
```diff
@@ -77,7 +84,19 @@ const VALID_PLATFORMS = [ 'stable', 'ptb', 'canary', 'dev', 'development' ];
...
     console.log('Try again with elevated permissions.');
+  } else if (e.code === 'ENOENT') {
+    console.log(
+      process.argv[2] === 'inject'
+        ? BasicMessages.PLUG_FAILED
+        : BasicMessages.UNPLUG_FAILED,
+      '\n'
+    );
+    console.log(
+      'Replugged wasn\'t able to inject itself due to missing files.',
```
Comment (asportnoy) [inline injectors/index.js:95]: Be more specific and more clear. Just say Discord <version> could not be found. Also, maybe make this yellow/bold.

File: injectors/index.js [hunk 3]
```diff
@@ -77,7 +84,19 @@ const VALID_PLATFORMS = [ 'stable', 'ptb', 'canary', 'dev', 'development' ];
...
+      '\n'
+    );
+    console.log('Try reinstalling Discord or Replugged.');
```
Comment (asportnoy) [inline injectors/index.js:98]: Tell the user to make sure that version is installed or to specify another version.

File: injectors/index.js [hunk 4]
```diff
@@ -77,7 +84,19 @@ const VALID_PLATFORMS = [ 'stable', 'ptb', 'canary', 'dev', 'development' ];
...
   } else {
-    console.error('fucky wucky', e);
+    console.error(`${AnsiEscapes.BOLD}${AnsiEscapes.RED}An unknown fucky wucky has occurred! Error info:${AnsiEscapes.RESET}\n`, e);
```
Comment (asportnoy) [inline injectors/index.js:100]: Do something like "Something went wrong". Keep the colors, makes it more clear.

**Review by asportnoy — CHANGES_REQUESTED** [review]

File: injectors/index.js (hunk 3 above)
Comment (spinfal) [reply injectors/index.js:98]: > Make sure that the specified platform is installed, or try again with a different platform.
How's this? I want to make sure that this one sounds clear enough.

Comment (asportnoy) [reply injectors/index.js:98]: Have it say the actual version instead of "specified platform". And give the syntax for using another platform.

**Review by lleyton — APPROVED** [review]
purrfect

**Pull request merged**